blocks/sketch/src/controls.js

```diff
@@ -90,20 +101,54 @@ const Controls = ( {
 				<ToolbarButton
 					icon={ trash }
 					onClick={ clear }
-					label={ __( 'Clear canvas', 'sketch' ) }
+					label={ __( 'Clear canvas', 'a8c-sketch' ) }
 					disabled={ isEmpty }
 				/>
+
+				<ToolbarButton
```

Comment:
Let's try adding a separate toolbar group for the upload action.

Reply:
addressed here https://github.com/Automattic/block-experiments/pull/283/commits/28a848344f358ed6612733f2c5bb652777fd8cce
<img width="614" alt="image" src="https://user-images.githubusercontent.com/77539/177018339-b37aa976-20aa-4eac-9cd0-bacd2ca9c614.png">